nbextensions/bulkImportCell/cellTabs.js

```diff
@@ -28,41 +34,46 @@ define([
          *          , etc.
          *       }
          */
-        constructor(options) {
-            this.bus = options.bus;
-            this.ui = options.ui;
-            this.tabToggleAction = options.toggleAction;
-            this.controlBarTabs = options.tabs;
-        }
+        let bus = options.bus,
+            ui,
```

Comment:
will this always be created by the `start` method, and never be passed in? Method docs should be updated as they still mention `ui`.

Reply:
Yup, fixed the docs, and rearranged where they go (above the function as usual).